gravitee-apim-console-webui/src/services-ngx/api-subscription-v2.service.ts

```diff
@@ -120,4 +120,24 @@ export class ApiSubscriptionV2Service {
       customApiKey,
     });
   }
+
+  revokeApiKey(apiId: string, subscriptionId: string, apiKeyId: string): Observable<ApiKey> {
+    return this.http.post<ApiKey>(
+      `${this.constants.env.v2BaseURL}/apis/${apiId}/subscriptions/${subscriptionId}/api-keys/${apiKeyId}/_revoke`,
+      {},
+    );
+  }
+
+  expireApiKey(apiId: string, subscriptionId: string, apiKeyId: string, expireAt: Date): Observable<ApiKey> {
+    return this.http.put<ApiKey>(`${this.constants.env.v2BaseURL}/apis/${apiId}/subscriptions/${subscriptionId}/api-keys/${apiKeyId}`, {
```

Comment:
Question: shouldn't this be a PATCH instead of a PUT ? cc @phiz71

Reply:
Actually, the expiration date is the only field you can change in an API Key, so PATCH would not be very useful here :p

Reply:
We'll keep the PUT for now 👍